tests/link/jax/signal/test_conv.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+import pytest
+
+from pytensor.tensor import dmatrix
+from pytensor.tensor.signal import convolve1d
+from tests.link.jax.test_basic import compare_jax_and_py
+
+
+@pytest.mark.parametrize("mode", ["full", "valid", "same"])
+def test_convolve1d(mode):
+    x = dmatrix("x")
+    y = dmatrix("y")
+    out = convolve1d(x[None], y[:, None], mode=mode)
+
+    rng = np.random.default_rng()
+    test_x = rng.normal(size=(3, 5))
```

Comment:
```suggestion
    test_x = rng.normal(size=(3, 5)).astype(floatX)
```

?

Reply:
Nope no stupid float32 in jax/numba tests and also I explicitly used dmatrix